src/App.js

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import Header from './components/Header';
+import Cards from './components/Cards';
+import Footer from './components/Footer';
+import Instruction from './components/Instruction';
+
+const App = () => {
+  const [cards, setCards] = useState([]);
+
+  //Delete Card
+  const deleteCard = (card) => {
+    var array = [...cards];
+    var index = array.indexOf(card);
+    if (index == -1) {
```

Comment:
no need to throw error. just write "return". If we use exception, whole app flow will stop.
And if your if statement is short one, you can write that beside if without curly braces